src/app/shared/components/Navbar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+interface NavbarProps {
+  title: string,
+}
+
+const NavBar = async (props: NavbarProps) => {
+  const { title } = props
+
+  return (
+    <div className="bg-cyan-700 h-[60px] w-full ">
+      <Link href="/">
```

Comment:
I've wrapped this Link around both the Logo & the Title. Not sure if we want this OR just wrapping it around the logo only.

Reply:
i would just wrap the Image, currently the whole navbar is clickable which feels a little strange

Reply:
Agreed & updated